src/components/OptimismDowntimeWarning/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Trans } from '@lingui/macro'
+import { SupportedChainId } from 'constants/chains'
+import { useActiveWeb3React } from 'hooks/web3'
+import { AlertOctagon } from 'react-feather'
+import styled from 'styled-components/macro'
+
+const Root = styled.div`
+  background-color: ${({ theme }) => theme.yellow3};
+  border-radius: 18px;
+  color: black;
+  margin-top: 16px;
+  max-width: 480px;
+  padding: 16px;
+  width: 100%;
+`
+const WarningIcon = styled(AlertOctagon)`
+  margin: 0 8px 0 0;
+`
+const TitleRow = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  margin: 0;
+  font-size: 20px;
+  font-weight: 600;
+  line-height: 25px;
+`
+const Body = styled.div`
```

Comment:
have you considered using `TYPE.body`? which may give you some default styling?

Reply:
I like this in principle, but it looks like the class specificity works such that I'd have to fill the style override w/ `!important`s to get it to work... :/

i.e. 
```
const Body = styled(TYPE.body)`
  font-size: 12px !important;
  line-height: 15px;
  margin: 8px 0 0 0 !important;
`
```

Is that the intended behavior of the `TYPE[text]` components? I didn't expect it to require that.

Reply:
hmm I think you are supposed to pass in font size and margin as component props

`<TYPE.body fontSize={12} margin='8px 0 0 0'>` or something similar. `line-height` can probably remain in the styled component.